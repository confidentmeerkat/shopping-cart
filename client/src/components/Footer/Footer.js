import React from 'react';
import '@styles/Footer.css';

const Footer = () => (
  <div className="footer">
    <p>MobileShop &copy; 2018</p>
  </div>
);

export default Footer;