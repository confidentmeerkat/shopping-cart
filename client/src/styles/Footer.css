.footer {
  width: 100%;
  height: auto;
  margin-top: -30px;
  background: #00BCD4;
  color: #c3e3e7;
  text-align: center;
}

.footer p {
  margin: 0;
  padding: 3px;
}