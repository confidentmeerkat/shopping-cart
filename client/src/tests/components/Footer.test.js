import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../components/Footer';

describe('<Footer />', () => {
  it('renders Footer correctly', () => {
    const wrapper = shallow(<Footer />);

    expect(wrapper).toMatchSnapshot();
  });
});