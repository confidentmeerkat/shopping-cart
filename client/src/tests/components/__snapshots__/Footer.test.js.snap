// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders Footer correctly 1`] = `
<div
  className="footer"
>
  <p>
    MobileShop Â© 2018
  </p>
</div>
`;
